Bindings/Java/tests/TestEditProperties.java

```diff
@@ -38,6 +38,23 @@ public static void main(String[] args) {
                     System.exit(-1);
                 } 
         }
+		// Now edit a Vec6 Property (inertia)
+		AbstractProperty inertiaProp = bdy.getPropertyByName("inertia");
+		Vec6 oldInertia = new Vec6();
+		for(int i=0; i<6; i++){
```

Comment:
I don't understand what this block is testing that the two blocks below don't already test: `set()` is called on L54, `getValueVec6()` is called on L53, and this block has no effect on the `assert` on L56.